lib/colorado_lottery.rb

```diff
@@ -0,0 +1,33 @@
+class ColoradoLottery
+  attr_reader :registered_contestants, :winners, :current_contestants
+  def initialize
+    @registered_contestants = {}
+    @winners = []
+    @current_contestants = {}
+  end
+
+  def interested_and_18?(name, game)
+    if name.age < 18
+      false
+    elsif !name.game_interests.include?(game.name)
+      false
+    else
+      true
+    end
+  end
+
+  def can_register?(name, game)
+    if interested_and_18?(name, game)
+      true
+    elsif !name.out_of_state? && game.national_drawing?
+      true
+    else
+      false
+    end
+  end
+
+  def register_contestant(contestant)
+    require "pry"; binding.pry
```

Comment:
We needed to build our hash here. I like seeing you reaching for pry, but try to remove these before submission.